### ISSUE TITLE
Make identify argument required

### ISSUE THREAD

**Issue by dgslos** [DominikBuchner/BOLDigger2#8]
Hi, 

Could you make of the identify argument an required argument to avoid this error? 

```
Traceback (most recent call last):
  File "/usr/local/bin/boldigger2", line 8, in <module>
    sys.exit(main())
  File "/usr/local/venvs/boldigger2/lib/python3.10/site-packages/boldigger2/__main__.py", line 66, in main
    thresholds.append(arguments.thresholds[i])
AttributeError: 'Namespace' object has no attribute 'thresholds'
```

**Comment by DominikBuchner** [comment]
Can you please also post the command to reproduce this error?
best
Dominik

**Comment by dgslos** [comment]
boldigger2 without arguments. Expected behavior is that it would mention there is at least on argument required or print the help.

**Comment by DominikBuchner** [comment]
fixed with 1.1.2
